request.go

```diff
@@ -31,7 +31,7 @@ func (c *Client) RawRequest(verb, p string, ro *RequestOptions) (*http.Request,
 
 	// Append the path to the URL.
 	u := *c.url
-	u.Path = path.Join(c.url.Path, p)
+	u.Path = strings.TrimRight(c.url.Path, "/") + "/" + strings.TrimLeft(p, "/")
```

Comment:
This preserves the current behavior _only at the join location_.  `p` remains unchanged.